### ISSUE TITLE
Is Key class supported?

### ISSUE THREAD

**Issue by AhmedSalahKhodair** [christianrondeau/SikuliSharp#21]
I can't send special keys by `Type` function such as `ENTER, TAB, ESC, SPACE` .

**Comment by christianrondeau** [comment]
Nope, but it should be pretty simple to add support for this! Let me know if you'd like information on this (make sure you can compile and run from a cloned version first)

**Comment by pcdigitacao2017** [comment]
why not use native commands.
just creating a method?

void type(string key)
{
SendKeys.Send(key);  // native
}

**Comment by christianrondeau** [comment]
@pcdigitacao2017 this is actually a good idea! However, it sounds like writing what Sikuli has been built for... it's probably as simple to pass through the key codes themselves (here: https://docs.oracle.com/javase/7/docs/api/constant-values.html#java.awt.event.KeyEvent.VK_TAB) by escaping them properly (and potentially creating a constants class, or using a builtin one if it matches).

Both solutions would work, I believe.
